test/games/strategy/triplea/delegate/StratBombTest.java

```diff
@@ -0,0 +1,127 @@
+package games.strategy.triplea.delegate;
+
+import static games.strategy.triplea.delegate.GameDataTestUtil.addTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.changefactory.ChangeFactory;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.ITestDelegateBridge;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Route;
+import games.strategy.engine.data.RouteScripted;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
+import games.strategy.engine.random.IRandomSource;
+import games.strategy.engine.random.ScriptedRandomSource;
+import games.strategy.test.TestUtil;
+import games.strategy.triplea.Constants;
+import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.attachments.UnitAttachment;
+import games.strategy.triplea.player.ITripleAPlayer;
+import games.strategy.triplea.xml.LoadGameUtil;
+
+public class StratBombTest {
+  private GameData m_data;
+
+  @Before
+  public void setUp() throws Exception {
+    m_data = LoadGameUtil.loadTestGame(LoadGameUtil.TestMapXml.GLOBAL1940);
+  }
+
+  private ITestDelegateBridge getDelegateBridge(final PlayerID player) {
+    return GameDataTestUtil.getDelegateBridge(player, m_data);
+  }
+
+
+  @Test
+  public void TestBombingRaid2targets() {
```

Comment:
I'd advocate that you test a minimum to show what you need - we have almost identical tests elsewhere and there little point to duplicate that test coverage (it is really not a good thing, you wind up in the state where one change breaks all tests, and not exactly one).

Regardless, the name of the method is a bit off:
- there are 3 targets here, not 2.
- we can describe that we are testing global allocation, something like `buildingsShootAtOnlyTheirBombers`, the method name that we are testing bombing targets is misleading since we are actually testing AA fire allocation.